### DISCUSSION TITLE
docs/user/overview/

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#2241] [Docs comments]
# docs/user/overview/

Manage users in Casdoor

https://casdoor.org/docs/user/overview/

**Comment by EliNaDeN** [comment]
Hi! Why can't the Admin of the organization change the "Name" of the users that he creates?

**Reply by hsluoyz** [reply]
@EliNaDeN can show provide a demo video to reproduce this issue?

**Comment by lucle1999** [comment]
Hi there, can I specify passwordType in the import file?

**Reply by hsluoyz** [reply]
Yes
